src/version/version.service.ts

```diff
@@ -0,0 +1,10 @@
+import { HttpService, Injectable } from '@nestjs/common';
+
+@Injectable()
+export class VersionService {
+  constructor(private httpService: HttpService) {}
+
+  getApi() {
+    return this.httpService.get('http://api.eddiehub.org');
```

Comment:
we could put the baseurl to a env-variable

Reply:
Great idea 👍